drivers/wireless/gs2200m.c

```diff
@@ -2108,6 +2152,35 @@ static enum pkt_type_e gs2200m_activate_wrx(FAR struct gs2200m_dev_s *dev,
   return gs2200m_send_cmd2(dev, cmd);
 }
 
+/****************************************************************************
+ * Name: gs2200m_powersave_wrx
+ * NOTE: See 9.1.1 Active Radio Receive
+ ****************************************************************************/
+
+static enum pkt_type_e gs2200m_powersave_wrx(FAR struct gs2200m_dev_s *dev,
+                                            uint8_t on)
+{
+  char cmd[30];
+
+  snprintf(cmd, sizeof(cmd), "AT+WRXPS=%d\r\n", on);
+  return gs2200m_send_cmd2(dev, cmd);
+}
+
+/****************************************************************************
+ * Name: gs2200m_syncloss
+ * NOTE: See 5.1.4 Sync loss interval
+ ****************************************************************************/
+
+static enum pkt_type_e gs2200m_syncloss(FAR struct gs2200m_dev_s *dev,
+                                            int val)
+{
+  char cmd[30];
+
+  printf("Sync interval = %d x 100ms\n", val);
```

Comment:
@takayoshi-k 
printf() should not be used in driver code.
Please replace it with wlinfo().

Reply:
@masayuki2009 
Thank you for reviewing.
I had forgotten remove it. I did it and force pushed.
Please check it again.